natlas-server/app/static/natlas.js

```diff
@@ -110,6 +111,23 @@ $(document).ready(function() {
     }
 });
 
+
+$(document).ready(function() {
```

Comment:
Throw this into some sort of login page specific TS file. `pages/login.ts` or similar